audit/README.md

```diff
@@ -0,0 +1,67 @@
+## Using the audit client
+
+```go
+package main
+
+import (
+        "net/http"
+        "fmt"
+        "time"
+        "github.com/philips-software/go-hsdp-api/audit"
+
+	dstu2cp "github.com/google/fhir/go/proto/google/fhir/proto/dstu2/codes_go_proto"
+	dstu2dt "github.com/google/fhir/go/proto/google/fhir/proto/dstu2/datatypes_go_proto"
+	dstu2pb "github.com/google/fhir/go/proto/google/fhir/proto/dstu2/resources_go_proto"
+)
+
+func main() {
+        client, err := audit.NewClient(http.DefaultClient, logging.Config{
+                SharedKey:    "YourSharedKeyHere=",
+                SharedSecret: "YourSharedSecretHere==",
+                BaseURL:      "https://audit-service.host.com",
+        })
+        if err != nil {
+            fmt.Printf("Error: %v\n", err)
+            return
+        }
+	timestamp := time.Date(2020, 12, 31, 23, 59, 59, 0, time.UTC)
+	event := &dstu2pb.AuditEvent{
```

Comment:
Probably a good idea to provide an `AuditEventTemplate` custom class that has values pre-set and can render a `dSTU2.AuditEvent` with required HSDP Audit fields set (e.g.: ProductKey extension)

Reply:
Similar to the [CDR helper stuff](https://pkg.go.dev/github.com/philips-software/go-hsdp-api@v0.28.0/cdr/helper/fhir/stu3)  I'm indeed planning to add some helper methods to construct common events. Are there any best practices e.g. for specific events, like `Login` / `Logout` that you know of? Any CCE examples? Maybe ILS community has expressed some opinions on this?

Reply:
The helper code is now there. As it gets used there will probably be some refinements along the way. I guess most folks are templating the AuditEvent JSON as it's quite gnarly to build up programmatically?! 😄 
In case there is an `OperationOutcome` it is returned in the STU3 namespace.